### DISCUSSION TITLE
Mediwiki markdown to GitHub markdown

### DISCUSSION THREAD

**Discussion by ceford** [max123kl/J4.docs#19] [General]
Last summer I wrote a script to populate a docusaurus installation with data from docs.joomla.org (it is online as a Developer Manual if anyone is interested - https://docus.jdocmanual.org/). This morning I revised that script to make it more general and I have uploaded a batch of Help pages.

I just noticed that I forgot to include the language folder. It is supposed to be like this

manuals/help/de/subfolders
manuals/help/en/subfolders
manuals/user/de/subfolders
manuals/user/en/subfolders

I think you should get the idea. At the moment only some of the en files have been uploaded. To have a look, go to my repo and switch to the cefxp branch and then select the manuals folder:

https://github.com/ceford/J4.docs 

or directly

https://github.com/ceford/J4.docs/tree/cefxp
https://github.com/ceford/J4.docs/tree/cefxp/manuals/help

Have a look at some the md files and let me know if you spot any problems.

I have left at the top of each file a few lines that are required by docusaurus. It illustrate an important point: if you are trying to keep translations in sync you need some sort of index.

I have a bug at the moment but when it is fixed I expect to be able to convert all of the help pages and their translations into gfm markdown. And all of the User tutorials too.

**Comment by max123kl** [comment]
Hi @ceford 
Short intermediate commentary:
I was writing a rather long comment with some tips for you and Alan. Due to a stupid mishap, I deleted it instead of saving it. I noticed a few things that might help you avoid mistakes in the future. I therefore need some more time to redo the whole thing.

**Comment by alann60** [comment]
That script sounds super useful. Beyond me!
